index.html

```diff
@@ -2,90 +2,96 @@
-          of Product & Engineering.
-        </p>
-      </div>
-    </div>
-    <div class="divider">Projects</div>
-    <div class="section projects">
-      <div class="project">
-        <div class="explanation">
-          <p>Animated floating Hal Jordan, the Green Lantern.</p>
-          <p>Image is an SVG created in Sketch.</p>
-          <p>Animation is achieved with CSS.</p>
-          <a href="https://github.com/jlcarmic/green-lantern" target="_blank">
-            <button class="button">Green Lantern</button>
-          </a>
-        </div>
-        <div class="screenshot">
-          <img src="images/greenlantern.png" />
-        </div>
-      </div>
 
-      <div class="project">
-        <div class="screenshot">
-          <img src="./images/producthuntalexa.png" />
-        </div>
-        <div class="explanation">
-          <p>An Alexa Skill for hearing Today's Featured products.</p>
-          <p>Built using NodeJS and AWS Lambda.</p>
-          <p>Utilizes ProductHunt's API.</p>
-          </p>
-          <a href="https://github.com/jlcarmic/producthunt_alexa_skill" target="_blank">
-            <button class="button">Today's Featured</button>
-          </a>
-        </div>
-      </div>
-
-      <div class="project">
-        <div class="explanation">
-          <p>An NPM package for accessing sports<br />stats via the Stattleship API.</p>
-          <p>Built in NodeJS and published to NPM<br />as an open source package.</p>
-          <a href="https://github.com/jlcarmic/node-stattleship" target="_blank">
-            <button class="button">Node Stattleship</button>
-          </a>
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+        <div class="container">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-nav-demo"
+                    aria-expanded="false">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+
+                <a href="#" class="navbar-brand" onclick="return false;"><i class="fa fa-home" aria-hidden="true"></i>
+                </a>
+
+
+            </div>
+
+            <div>
+
+                <ul class="nav navbar-nav">
+                    <li class="active"><a href="index.html">Home</a></li>
+                    <li class="About me"><a href="MyWork.html">My Work</a></li>
+                </ul>
+
+                <ul class="nav navbar-nav navbar-right">
+                    <li>
+                        <p id="phone">(732)-991-8984</p>
+                    </li>
+                    <li>
+                        <p id="git"><i class="fa fa-github" aria-hidden="true"></i><a
+                                href="https://github.com/Bsilvs95">Git</a></p>
```

Comment:
Generally when you link to an external site you should use `target="_blank"` so it throws in a new tab. Links within your site it is ok to do in the same tab.